src/mount/mount-tool.c

```diff
@@ -598,6 +610,58 @@ static int start_transient_mount(
         return 0;
 }
 
+static int action_remount(
+                sd_bus *bus,
+                int argc,
+                char **argv) {
+
+        _cleanup_(sd_bus_message_unrefp) sd_bus_message *m = NULL, *reply = NULL;
+        _cleanup_(sd_bus_error_free) sd_bus_error error = SD_BUS_ERROR_NULL;
+        _cleanup_free_ char *mount_unit = NULL;
+        _cleanup_free_ char *object_path = NULL;
+        int r;
+
+        r = unit_name_from_path(argv[optind], ".mount", &mount_unit);
+        if (r < 0)
+                return log_error_errno(r, "Failed to make mount unit name: %m");
+
+        object_path = unit_dbus_path_from_name(mount_unit);
+        log_debug("object_path %s", object_path);
```

Comment:
I do not think it is necessary... Of course, it is in debug level though...

Reply:
Right, I forgot to remove it. Also `unit_dbus_path_from_name()` can also return NULL in case of no memory. I removed the debug call and added a check for NULL. PTAL.